xarray/tests/test_cftime_offsets.py

```diff
@@ -445,7 +445,6 @@ def test_eq(a, b):
     (Second(), 3, Second(n=3)),
     (Millisecond(), 3, Millisecond(n=3)),
     (Microsecond(), 3, Microsecond(n=3)),
-    (Day(), 0.5, Hour(n=12)),
```

Comment:
Now that `Day` is non-`Tick`-like, multiplying by a `float` is no longer valid in pandas:
```
>>> import pandas as pd
>>> 2.0 * pd.tseries.frequencies.Day(2)
Traceback (most recent call last):
  File "<python-input-2>", line 1, in <module>
    2.0 * pd.tseries.frequencies.Day(2)
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
TypeError: unsupported operand type(s) for *: 'float' and 'pandas._libs.tslibs.offsets.Day'
```
A similar error is raised now for the cftime version of this offset.